redis/client.py

```diff
@@ -1675,6 +1683,64 @@ def sunionstore(self, dest, keys, *args):
         args = list_or_args(keys, args)
         return self.execute_command('SUNIONSTORE', dest, *args)
 
+    # TIMESERIES COMMANDS
+    def tsadd(self, name, *args, **kwargs):
+        """
+        Set any number of timestamp, element pairs to the timeseries ``name``. Pairs
+        can be specified in two ways:
+
+        As *args, in the form of: timestamp1, name1, timestamp2, name2, ...
+        or as **kwargs, in the form of: name1=timestamp1, name2=timestamp2, ...
+
+        The following example would add four values to the 'my-key' key:
+        redis.tsadd('my-key', 11, 'name1', 22, 'name2', name3=33, name4=44)
+        """
+        pieces = []
+        if args:
+            if len(args) % 2 != 0:
+                raise RedisError("TSADD requires an equal number of "
+                                 "values and scores")
+            pieces.extend(args)
+        for pair in iteritems(kwargs):
+            pieces.append(pair[1])
+            pieces.append(pair[0])
+        return self.execute_command('TSADD', name, *pieces)
+
+    def tsget(self, name, ts):
+        "Return the element corresponding to the specified timestamp `ts` in the time series``name``"
```

Comment:
Might be good to keep the same comment style - use """ encompassing the contents.